app/views/tickets/new.html.erb

```diff
@@ -54,7 +54,7 @@
               <span class="error-message">Enter a name</span>
             <% end %>
           </label>
-          <%= f.text_area(:name,
+          <%= f.text_field(:name,
                           class: input_box_class_for(@ticket, :name),
                           id: "example-name",
                           rows: 1 ) %>
```

Comment:
We should drop the rows attribute too.

Reply:
yep, good spot

Reply:
@andrewgarner removed the row attribute 👍